.openpublishing.redirection.json

```diff
@@ -2232,6 +2232,26 @@
             "source_path": "docs/visual-basic/reference/vb6-support.md",
             "redirect_url": "/visualstudio/vb6/vb6-support",
             "redirect_document_id": true
+        },
+        {
```

Comment:
let's move this block to line 768 since we try to keep things in alphabetical order